config/job.js

```diff
@@ -180,6 +186,8 @@ const SCHEMA_JOB = Joi.object()
         description: SCHEMA_DESCRIPTION,
         environment: SCHEMA_ENVIRONMENT,
         freezeWindows: SCHEMA_FREEZEWINDOWS,
+        blockedBySameJob: SCHEMA_BLOCKEDBYSAMEJOB,
+        blockedBySameJobWaitTime: SCHEMA_BLOCKEDBYSAMEJOBWAITTIME,
```

Comment:
it kind of seems weird to me that it's a job-level field vs annotation, but I guess `blockedBy` is also at this level

I think I would prefer these fields as annotations

Reply:
Reasoning is that you probably wouldn't want this on every job

Reply:
could be job-level annotation, right?